server/apps/immich/src/api-v1/asset/asset-repository.ts

```diff
@@ -343,4 +344,13 @@ export class AssetRepository implements IAssetRepository {
     });
     return new CheckExistingAssetsResponseDto(existingAssets.map((a) => a.deviceAssetId));
   }
+
+  async countByIdAndUser(assetId: string, userId: string): Promise<number> {
+    return await this.assetRepository.count({
```

Comment:
is the `await` necessary in this line? I hope `assetRepository.count(...)` already returns a `Promise<number>`.

Reply:
I did it for the convention because we do it like this in most methods.